samples/sampler/stories/default/MediaPlayer.js

```diff
@@ -3,17 +3,29 @@ import {storiesOf} from '@storybook/react';
 
 import MediaPlayer from '@enact/agate/MediaPlayer';
 
+const audioFiles = [
+	'https://file-examples-com.github.io/uploads/2017/11/file_example_MP3_700KB.mp3',
+	'https://www.soundhelix.com/examples/mp3/SoundHelix-Song-8.mp3',
+	'https://www.soundhelix.com/examples/mp3/SoundHelix-Song-9.mp3',
+	'https://www.soundhelix.com/examples/mp3/SoundHelix-Song-10.mp3',
+	'https://www.soundhelix.com/examples/mp3/SoundHelix-Song-16.mp3'
```

Comment:
Are these files license free?
Hope we don't get in trouble with the license.

Reply:
I will replace them with some that are surely license free. Thank you!